### PR TITLE
Fix SDK version

### PR DESCRIPTION
**Pull request by yuwzho**
##### ISSUE TYPE

 - Bugfix Pull Request

##### ANSIBLE VERSION

```
2.4.3
```

##### ADDITIONAL INFORMATION

```

```

### CODE REVIEW

**Comment by mattclay** [comment]
CI failure in integration tests due to tracebacks:

```
Traceback (most recent call last):
  File "/tmp/ansible_dQHjFx/ansible_module_azure_rm_availabilityset.py", line 322, in <module>
    main()
  File "/tmp/ansible_dQHjFx/ansible_module_azure_rm_availabilityset.py", line 319, in main
    AzureRMAvailabilitySet()
  File "/tmp/ansible_dQHjFx/ansible_module_azure_rm_availabilityset.py", line 195, in __init__
    supports_tags=True)
  File "/tmp/ansible_dQHjFx/ansible_modlib.zip/ansible/module_utils/azure_rm_common.py", line 358, in __init__
  File "/tmp/ansible_dQHjFx/ansible_module_azure_rm_availabilityset.py", line 205, in exec_module
    self.models = self.client.availability_sets.models
AttributeError: 'AvailabilitySetsOperations' object has no attribute 'models'
```

```
Traceback (most recent call last):
  File "/tmp/ansible_dsYZqH/ansible_module_azure_rm_virtualmachine.py", line 1700, in <module>
    main()
  File "/tmp/ansible_dsYZqH/ansible_module_azure_rm_virtualmachine.py", line 1696, in main
    AzureRMVirtualMachine()
  File "/tmp/ansible_dsYZqH/ansible_module_azure_rm_virtualmachine.py", line 706, in __init__
    self.network_models = None
AttributeError: can't set attribute
```

File: test/sanity/validate-modules/ignore.txt [hunk 1]
```diff
@@ -444,10 +444,7 @@ lib/ansible/modules/cloud/azure/azure_rm_deployment.py E322
 lib/ansible/modules/cloud/azure/azure_rm_deployment.py E324
 lib/ansible/modules/cloud/azure/azure_rm_deployment.py E325
 lib/ansible/modules/cloud/azure/azure_rm_deployment.py E326
-lib/ansible/modules/cloud/azure/azure_rm_dnsrecordset.py E322
-lib/ansible/modules/cloud/azure/azure_rm_dnsrecordset.py E324
-lib/ansible/modules/cloud/azure/azure_rm_dnsrecordset.py E325
-lib/ansible/modules/cloud/azure/azure_rm_dnsrecordset.py E326
+lib/ansible/modules/cloud/azure/azure_rm_dnsrecordset.py E321
```
Comment (mattclay) [inline test/sanity/validate-modules/ignore.txt:447]: PRs should not add ignores. The underlying issue needs to be addressed.

Comment (yuwzho) [reply test/sanity/validate-modules/ignore.txt:447]: So how can I address the issue? And the sanity test? I am really confused by this.

Comment (mattclay) [reply test/sanity/validate-modules/ignore.txt:447]: I'd start by working down the list of import errors. The first one I see is:

> AttributeError: 'AzureRMRecordSet' object has no attribute '_dns_client'

Which is reported as being called from here:

https://github.com/ansible/ansible/pull/35823/files#diff-e448c30ab815c0d2e05412e35bc7da19R245

The issue appears to be that the base class constructor hasn't been called yet, so `_dns_client` hasn't been initialized yet:

https://github.com/VSChina/ansible/blob/367d66b8a67a568fe3a02dc6cee36f5c033728d3/lib/ansible/module_utils/azure_rm_common.py#L260

Comment (mattclay) [reply test/sanity/validate-modules/ignore.txt:447]: You may find it easier to try fixing the integration test failures first:

https://app.shippable.com/github/ansible/ansible/runs/56819/62/tests
https://app.shippable.com/github/ansible/ansible/runs/56819/68/tests

**Comment by nitzmahone** [comment]
Finally got around to looking at this in more detail- looks like the conversion is incomplete, so the import errors are actually legitimate (as we'd hit the same errors when running). I've started working on the necessary fixes, will return to it on Monday.

**Pull request closed without merging**